applications/serial_lte_modem/doc/TCPIP_AT_commands.rst

```diff
@@ -203,21 +162,58 @@ Response syntax
 
 ::
 
-   #XSOCKET: <list of op value>,<list of type value>,<list of roles>,<sec-tag>
+   #XSOCKET: <list of op value>,<list of type value>,<list of roles>
 
 
 * The ``<op>`` parameter can accept one of the following values:
 
   * ``0`` - Open
   * ``1`` - Close
 
-* The ``<type>`` parameter value depends on the presence of the <sec_tag> parameter.
-  When the ``<sec_tag>`` is not specified:
+* The ``<type>`` parameter can accept one of the following values:
 
   * 1: SOCK_STREAM for TCP
   * 2: SOCK_DGRAM for UDP
 
-  When the ``<sec_tag>`` is specified:
+* The ``<role>`` parameter can accept one of the following values:
+
+  * ``0`` - Client
+  * ``1`` - Server
+
+
+Examples
+~~~~~~~~
+
+::
+
+   AT#XSOCKET=?
+   #XSOCKET: (0,1),(1,2)
+   OK
+
+Secure Socket #XSSOCKET
+=======================
+
+The ``#XSSOCKET`` command allows you to open or close a secure socket, or to check the socket handle.
```

Comment:
> Normal socket and secure socket share the close function so no need of change.

In such case this documentation is not correct, as you don't use `#XSSOCKET` to close a socket.

Reply:
I mean that AT#XSOCKET=0 and AT#XSSOCET=0 shares the same do_socket_close() and will have the same "#XSOCKET: 0,\"closed\"" command response. Normal and secure socket only differs when opening.